spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/config/KafkaMessageChannelBinderConfiguration.java

```diff
@@ -33,9 +34,13 @@
 @ConfigurationProperties(prefix = "spring.cloud.stream.binder.kafka")
 public class KafkaMessageChannelBinderConfiguration {
 
-	private String zkAddress;
+	private String[] zkNodes;
 
-	private String brokers;
+	private String zkDefaultPort;
+
+	private String[] brokers;
+
+	private String brokersDefaultPort;
```

Comment:
one final nitpick: I think I would change this to `defaultBrokerPort` and the corresponding ZK property name to `defaultZkPort`... to me that reads better and avoids the singular vs. plural situation with "brokers" currently